airbrakes/state.py

```diff
@@ -146,10 +146,12 @@ def update(self):
 
         # if our prediction is overshooting our target altitude, extend the airbrakes
         if self.context.apogee_predictor.apogee > TARGET_ALTITUDE:
-            self.context.extend_airbrakes()
-        else:
-            # If our prediction is not overshooting, retract the airbrakes
+            if not self.airbrakes_extended:
+                self.context.extend_airbrakes()
+                self.airbrakes_extended = True
+        elif self.airbrakes_extended:
             self.context.retract_airbrakes()
```

Comment:
should check `if self.airbrakes_extended`

Reply:
done by ruff, it's correct

Reply:
lol am i just blind